Framework/PhysicsSupport/include/Framework/O2DatabasePDG.h

```diff
@@ -0,0 +1,26 @@
+// Copyright 2019-2020 CERN and copyright holders of ALICE O2.
```

Comment:
This would lead to 2 header files of the same name `O2DatabasePDG.h` in O2. Maybe a better name could be found? (O2DatabasePDGPlugin ... or similar)?

Reply:
ok.